### PR TITLE
i18n(fr): Update components/i18n/sidebar/configuration/frontmatter.mdx files

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description

Update some French files : 
- [i18n(fr): Update](https://github.com/withastro/starlight/commit/72ec8d1a5e28e225fdead4580b44d1e50105cf63) [guides/components.mdx](https://github.com/withastro/starlight/commit/72ec8d1a5e28e225fdead4580b44d1e50105cf63) [from](https://github.com/withastro/starlight/commit/72ec8d1a5e28e225fdead4580b44d1e50105cf63) https://github.com/withastro/starlight/pull/1530
- [i18n(fr): Update](https://github.com/withastro/starlight/commit/39cae206ec4c55999ec692f52cff1bdf1c6a1797) [guides/i18n.mdx](https://github.com/withastro/starlight/commit/39cae206ec4c55999ec692f52cff1bdf1c6a1797) [from](https://github.com/withastro/starlight/commit/39cae206ec4c55999ec692f52cff1bdf1c6a1797) https://github.com/withastro/starlight/pull/1841
- [i18n(fr): Update](https://github.com/withastro/starlight/commit/bbc3fad8cebc1193633b6abc454fa25103b00364) [guides/sidebar.mdx](https://github.com/withastro/starlight/commit/bbc3fad8cebc1193633b6abc454fa25103b00364) [from](https://github.com/withastro/starlight/commit/bbc3fad8cebc1193633b6abc454fa25103b00364) https://github.com/withastro/starlight/pull/1530
- [i18n(fr): Update](https://github.com/withastro/starlight/commit/83e833f6af24e2738e7c64bb8c0f2f1f9d25fdf6) [reference/configuration.mdx](https://github.com/withastro/starlight/commit/83e833f6af24e2738e7c64bb8c0f2f1f9d25fdf6) [from](https://github.com/withastro/starlight/commit/83e833f6af24e2738e7c64bb8c0f2f1f9d25fdf6) https://github.com/withastro/starlight/pull/1958
- [i18n(fr): Update](https://github.com/withastro/starlight/commit/14ba275339f132a8925bbd0523abfac3e0aaa468) [reference/configuration.mdx](https://github.com/withastro/starlight/commit/14ba275339f132a8925bbd0523abfac3e0aaa468) [from](https://github.com/withastro/starlight/commit/14ba275339f132a8925bbd0523abfac3e0aaa468) https://github.com/withastro/starlight/pull/1530
- [i18n(fr): Update](https://github.com/withastro/starlight/commit/3fd606d7dcb3ec6d2f24d3351985d2959406bec3) [reference/frontmatter.mdx](https://github.com/withastro/starlight/commit/3fd606d7dcb3ec6d2f24d3351985d2959406bec3) [from](https://github.com/withastro/starlight/commit/3fd606d7dcb3ec6d2f24d3351985d2959406bec3) https://github.com/withastro/starlight/pull/1530

### CODE REVIEW

**Review by dreyfus92 — APPROVED** [review]
Merci @thomasbnt, LGTM ✅

File: docs/src/content/docs/fr/guides/components.mdx [hunk 1]
```diff
@@ -423,6 +422,36 @@ import { Steps } from '@astrojs/starlight/components';
...
 
+### Badges
+
+import { Badge } from '@astrojs/starlight/components';
+
+Utilisez le composant `<Badge>` pour afficher de petits éléments d'information, tels que des statuts ou des étiquettes.
+
+Passez le contenu que vous souhaitez afficher à l'attribut `text` du composant `<Badge>`.
+
+Par défaut, le badge utilise la couleur d'accentuation du thème de votre site. Pour utiliser une couleur de badge intégrée, définissez l'attribut `variant` à l'une des valeurs suivantes : `note` (bleu), `tip` (violet), `danger` (rouge), `caution` (orange), ou `success` (vert).
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/guides/components.mdx:433]:
```suggestion
Par défaut, le badge utilisera la couleur d'accentuation du thème de votre site. Pour utiliser une couleur de badge intégrée, définissez l'attribut `variant` à l'une des valeurs suivantes : `note` (bleu), `tip` (violet), `danger` (rouge), `caution` (orange), ou `success` (vert).
```

To match the English version.

File: docs/src/content/docs/fr/guides/components.mdx [hunk 2]
```diff
@@ -423,6 +422,36 @@ import { Steps } from '@astrojs/starlight/components';
...
+
+L'attribut `size` (par défaut : `small`) contrôle la taille du texte du badge. Les attributs `medium` et `large` sont également disponibles pour afficher un badge plus grand.
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/guides/components.mdx:435]:
```suggestion
L'attribut `size` (par défaut : `small`) contrôle la taille du texte du badge. Les valeurs `medium` et `large` sont également disponibles pour afficher un badge plus grand.
```

To make sure it's obvious these are not other props but valid values for `size`.

File: docs/src/content/docs/fr/guides/components.mdx [hunk 3]
```diff
@@ -423,6 +422,36 @@ import { Steps } from '@astrojs/starlight/components';
...
+
+Pour une personnalisation plus poussée, utilisez d'autres attributs `<span>` tels que `class` ou `style` avec du CSS personnalisé.
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/guides/components.mdx:437]:
```suggestion
Pour une personnalisation plus poussée, utilisez d'autres attributs de  l'élément `<span>` tels que `class` ou `style` avec du CSS personnalisé.
```

Not sure this reads as well in French as in English so I think adding a bit more context makes sense here.

File: docs/src/content/docs/fr/guides/components.mdx [hunk 4]
```diff
@@ -423,6 +422,36 @@ import { Steps } from '@astrojs/starlight/components';
...
+
+```mdx title="src/content/docs/example.mdx"
+import { Badge } from '@astrojs/starlight/components';
+
+<Badge text="Nouveau" variant="tip" size="small" />
+<Badge text="Déclassé" variant="caution" size="medium" />
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/guides/components.mdx:443]:
```suggestion
<Badge text="Obsolète" variant="caution" size="medium" />
```

Is "Déclassé" something you have encountered? I've never heard it so I took the suggested word in development from [here](https://fr.wiktionary.org/wiki/deprecated):

> (Informatique) Obsolète, périmé.

File: docs/src/content/docs/fr/guides/i18n.mdx [hunk 5]
```diff
@@ -67,6 +67,8 @@ Starlight offre une prise en charge intégrée des sites multilingues, y compris
 
 </Steps>
 
+Pour les scénarios i18n plus avancés, Starlight prend également en charge la configuration de l'internationalisation à l'aide de l'option de [configuration `i18n` dans Astro](https://docs.astro.build/fr/guides/internationalization/#configure-i18n-routing).
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/guides/i18n.mdx:70]:
```suggestion
Pour les scénarios i18n plus avancés, Starlight prend également en compte la configuration de l'internationalisation à l'aide de l'option de [configuration `i18n` d'Astro](https://docs.astro.build/fr/guides/internationalization/#configure-i18n-routing).
```

I think ["prendre en compte"](https://fr.wiktionary.org/wiki/prendre_en_compte) is closer to what is happening under the hood in this case compared to ["prendre en charge"](https://fr.wiktionary.org/wiki/prendre_en_charge).

File: docs/src/content/docs/fr/guides/i18n.mdx [hunk 6]
```diff
@@ -271,3 +273,16 @@ export const collections = {
 ```
 
 Consultez [« Définir un schéma de collection de contenus »](https://docs.astro.build/fr/guides/content-collections/#defining-a-collection-schema) dans la documentation d'Astro pour en savoir plus sur les schémas de collection de contenus.
+
+## Accès à la langue locale actuelle
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/guides/i18n.mdx:277]:
```suggestion
## Accès aux paramètres régionaux
```

[Reference](https://fr.wikipedia.org/wiki/Param%C3%A8tres_r%C3%A9gionaux): looks like this is the proper word for "locales" which is an anglicism in French.

File: docs/src/content/docs/fr/guides/i18n.mdx [hunk 7]
```diff
@@ -271,3 +273,16 @@ export const collections = {
...
+
+Vous pouvez utiliser [`Astro.currentLocale`](https://docs.astro.build/fr/reference/api-reference/#astrocurrentlocale) pour lire les paramètres linguistiques actuels dans les composants `.astro`.
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/guides/i18n.mdx:279]:
```suggestion
Vous pouvez utiliser [`Astro.currentLocale`](https://docs.astro.build/fr/reference/api-reference/#astrocurrentlocale) pour lire les paramètres régionaux et linguistiques actuels dans les composants `.astro`.
```

Using the same reference, to emphasize this is not only the language:

> Language refers to a system of communication, such as English, Spanish, or Mandarin, while locale encompasses not only the language but also the region […}

File: docs/src/content/docs/fr/guides/i18n.mdx [hunk 8]
```diff
@@ -271,3 +273,16 @@ export const collections = {
...
+
+L'exemple suivant lit la langue locale courante et l'utilise pour générer un lien vers une page d'information dans la langue courante :
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/guides/i18n.mdx:281]:
```suggestion
L'exemple suivant lit les paramètres régionaux actuels et les utilise pour générer un lien vers une page d'information dans la même langue :
```

File: docs/src/content/docs/fr/guides/sidebar.mdx [hunk 9]
```diff
@@ -295,13 +295,14 @@ La configuration ci-dessus génère la barre latérale suivante :
...
+### Variantes de badges et style personnalisé
 
-Personnalisez le style du badge en utilisant un objet avec les propriétés `text` et `variant`.
+Personnalisez le style du badge en utilisant un objet avec les propriétés `text`, `variant`, et `class`.
 
 La propriété `text` représente le contenu à afficher (par exemple, "Nouveau").
-Remplacez le style `default`, qui utilise la couleur d'accentuation de votre site, en définissant la propriété `variant` avec l'une des valeurs suivantes : `note`, `tip`, `danger`, `caution` ou `success`.
+Par défaut, le badge utilise la couleur d'accentuation de votre site. Pour utiliser un style de badge intégré, définissez la propriété `variant` à l'une des valeurs suivantes : `note`, `tip`, `danger`, `caution` ou `success`.
 
+En option, vous pouvez créer un style de badge personnalisé en définissant la propriété `class` à un nom de classe CSS.
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/guides/sidebar.mdx:305]:
```suggestion
Facultativement, vous pouvez créer un style de badge personnalisé en définissant la propriété `class` à un nom de classe CSS.

```

File: docs/src/content/docs/fr/reference/configuration.mdx [hunk 10]
```diff
@@ -568,3 +569,13 @@ starlight({
 ```
 
 Consultez la [référence des modules d'extension](/fr/reference/plugins/) pour plus de détails pour créer vos propres modules d'extension.
+
+### `credits`
+
+Permettre l'affichage d'un lien "Built with Starlight" dans le pied de page de votre site.
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/reference/configuration.mdx:575]:
```suggestion
Permet l'affichage d'un lien "Créé avec Starlight" dans le pied de page de votre site.
```

The feature was [translated](https://github.com/withastro/starlight/blob/main/packages/starlight/translations/fr.json#L31) live during Starlight birthday as there was multiple people speaking French in the call so we can safely use that.

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for all the updates :raised_hands:

Left a few minor suggestions/questions but this is looking really great!

File: docs/src/content/docs/fr/guides/i18n.mdx [hunk 11]
```diff
@@ -271,3 +273,16 @@ export const collections = {
...
+
+```astro
+---
+// src/components/AboutLink.astro
+---
+<a href={`/${Astro.currentLocale}/about`}>About</a>
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/guides/i18n.mdx:287]:
```suggestion
<a href={`/${Astro.currentLocale}/about`}>À propos</a>
```

Forgot about the link text during my review.

File: docs/src/content/docs/fr/reference/configuration.mdx (hunk 10 above)
Comment (thomasbnt) [reply docs/src/content/docs/fr/reference/configuration.mdx:575]: Oh of course!

File: docs/src/content/docs/fr/guides/components.mdx (hunk 4 above)
Comment (thomasbnt) [reply docs/src/content/docs/fr/guides/components.mdx:443]: My bad, yeah I using the same word but at this moment I didn't know what he meant in French, being used to writing and using it in English.

**Comment by thomasbnt** [comment]
Hello and thanks both @dreyfus92 @HiDeoo for the review! I added your suggestions, thanks a lot!

**Review by pioupia — APPROVED** [review]
LGTM!

**Review by HiDeoo — APPROVED** [review]
Thanks for the fast updates and everyone for the reviews, let's ship all this new content :rocket:

**Pull request merged**